tests/e2e/models/mnist_train_test/generate_test_data.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import jax
```

Comment:
Can you add a requirements.txt file for running this script? I haven't run it successfully yet

Reply:
Done.

Reply:
Did you forget to `git add`?

Reply:
Oh I see it's "generate_test_data_requirements.txt" not "requirements.txt". The name gets cut off in the GitHub UI

Reply:
I don't believe that requirements file is sufficient though

```python
$ pip install -r tests/e2e/models/mnist_train_test/generate_test_data_requirements.txt
ERROR: Could not find a version that satisfies the requirement iree-jax (from versions: none)
ERROR: No matching distribution found for iree-jax
```

Ok, so iree-jax is actually not pip installable anyway and there's not a requirements file way around that. Following the iree-jax install instructions in a clean venv though, this still doesn't work:

```
Traceback (most recent call last):
  File "/usr/local/google/home/gcmn/src/iree/tests/e2e/models/mnist_train_test/generate_test_data.py", line 17, in <module>
    from examples import datasets
ModuleNotFoundError: No module named 'examples'
```

Please add a comment linking to https://github.com/iree-org/iree-jax#development and then provide a functioning requirements file for the rest of the requirements.

Reply:
I fixed the requirements.
I have not noticed that the logic to download the dataset is is not included in the `iree-jax` package. I ended up copying it from there. Maybe we can remove it here if it gets included into some package in IREE JAX.